conf/messages.en

```diff
@@ -253,28 +253,28 @@ calc.whatToDoNext.noGain.bulletTitle = If you have made a loss that you would li
 calc.whatToDoNext.noGain.bulletPointTwo = report it now using our online form
 calc.whatToDoNext.noGain.reportImportant = If you report it now, you will still need to declare your capital gains through Self Assessment.
-calc.whatToDoNext.noGain.reportNowInfo = To report now you will need a Government Gateway account. If you do not already have an account, you can get one before you sign in.
+calc.whatToDoNext.noGain.reportNowInfo = To report now, you''ll need HMRC sign in details. If you do not already have these, you can create them before you sign in.
 
 calc.whatToDoNext.gain.bulletTitle = You can either:
 calc.whatToDoNext.gain.bulletPointOne = report and pay on your Self Assessment return for the {0} tax year
 calc.whatToDoNext.gain.bulletPointTwo = report and pay now using our online form
 calc.whatToDoNext.gain.reportImportant = If you report and pay now, you will still need to declare your capital gains through Self Assessment.
-calc.whatToDoNext.gain.reportNowInfo = To report now you will need a Government Gateway account. If you do not already have an account, you can get one before you sign in.
+calc.whatToDoNext.gain.reportNowInfo = To report now, you''ll need HMRC sign in details. If you do not already have these, you can create them before you sign in.
 calc.whatToDoNext.exitSurveyMessage = If you do not wish to report capital gains at this time, you can exit now or
 calc.whatToDoNext.exitSurveyLink = leave feedback for the CGT calculator service (takes 2 minutes)
 ## What Next Non-SA Gain messages
 calc.resident.whatNextNonSa.gain.title = What happens next
 calc.resident.whatNextNonSa.gain.reportNow = Report now
 calc.resident.whatNextNonSa.gain.govUk = Return to GOV.UK
 calc.resident.whatNextNonSa.gain.detailsOne = Use our online form to report and pay your Capital Gains Tax.
-calc.resident.whatNextNonSa.gain.detailsTwo = To report now you will need a Government Gateway account. If you do not already have an account, you can get one before you sign in.
+calc.resident.whatNextNonSa.gain.detailsTwo = To report now, you''ll need HMRC sign in details. If you do not already have these, you can create them before you sign in.
 
 ## What Next Non-SA Loss messages
 calc.resident.whatNextNonSa.loss.title = What happens next
 calc.resident.whatNextNonSa.loss.reportNow = Report now
 calc.resident.whatNextNonSa.loss.govUk = Return to GOV.UK
 calc.resident.whatNextNonSa.loss.detailsOne = If you have made a loss that you would like to use against any future Capital Gains Tax, you can report it using our online form.
-calc.resident.whatNextNonSa.loss.detailsTwo = To report now you will need a Government Gateway account. If you do not already have an account, you can get one before you sign in.
+calc.resident.whatNextNonSa.loss.detailsTwo = To report now, you''ll need HMRC sign in details. If you do not already have these, you can create them before you sign in.
```

Comment:
Same here, make sure on your local machine that we can only see one apostrophe. Thanks